packages/react-native-win32/src/Libraries/Components/View/ReactNativeViewViewConfig.win32.js

```diff
@@ -134,10 +134,13 @@ const ReactNativeViewConfig = {
   validAttributes: {
     ...ReactNativeViewViewConfigAndroid.validAttributes,
     accessibilityActions: true,
+    accessibilityDescription: true,
```

Comment:
I have Win32 specific props at the bottom of this list and commented to avoid merge issues. Could you move things there?